src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/PackTask.cs

```diff
@@ -75,6 +77,14 @@ public IPackTaskLogic PackTaskLogic
 
         public override bool Execute()
         {
+#if DEBUG
```

Comment:
oh i put it for testing purposes and forgot to remove it. thanks for catching this.